cn_indian_payroll/cn_indian_payroll/overrides/salary_structure_assignment.py

```
@@ -86,34 +57,17 @@ def update_employee_promotion(self):
 
 
 
-
-
-
-
-    def insert_tax_declaration(self):
-        
+    def insert_tax_declaration_list(self):
```

Ruff (0.8.2)

`insert_tax_declaration_list` is too complex (21 > 10)

(C901)